src/api/emotes.md

```diff
@@ -0,0 +1,72 @@
+---
+id: player_emotes
+name: Player Emotes
+title: Player Emotes
+tags:
+    - API
+    - Reference
+---
+
+# Player Emotes
+
+<!-- TODO: Add Intro -->
```

Comment:
Player emotes are the animations like dances and gestures that players  are able to start at will through the **EMOTES** menu, usually by pressing ++B++ or the middle mouse button. Creators can find out what emote a player is currently playing through the ``activeEmote`` property on each player. See the Player type for more information.

Not all players have access to all emotes, but can purchase them through the Shop and unlock them through quests. The list below shows all the possible emotes that are currently available on Core.

Reply:
sorry,, edit:

> ... through the ``activeEmote`` property and the ``emoteStartedEvent`` and ``emoteStoppedEvent`` on the player.